go/main.go

```diff
@@ -0,0 +1,217 @@
+package main
```

Comment:
You've checked `go/cbox` into this branch -- Github says it's a binary. Perhaps that was by accident?

Reply:
I uses cgo to test  if this ffi  lib  works well

Reply:
Sure, but I still don't understand why there's a 2.49 MB binary in the repository :slightly_smiling_face:

Reply:
i thought you mean main.go this is not a  binay  and cbox is, i have delete now  ,thanks for the check👍

Reply:
Oh, sorry for the confusion -- I would've commented on `cbox` but Github doesn't let me :unamused:

Reply:
ok i see